CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+##[0.18.23](https://github.com/brave/browser-laptop/releases/tag/v0.18.23dev)
+ - Fixed spell check not working on Disqus comment boxes. ([#10040](https://github.com/brave/browser-laptop/issues/10040))
+ - Fixed findbar slowness (cmd + F, typing). ([#10271](https://github.com/brave/browser-laptop/issues/10271))
+ - Fixed issue where pinned tab can't be really unpinned. ([#10241](https://github.com/brave/browser-laptop/issues/10241))
+ - Fixed pinned tab will not display. ([#10122](https://github.com/brave/browser-laptop/issues/10122))
+ - Fixed cursor not shown when editing a bookmark. ([#10104](https://github.com/brave/browser-laptop/issues/10104))
+ - Fixed short term video freezes. ([#9808](https://github.com/brave/browser-laptop/issues/9808))
+ - Fixed browser freezes per 5 minutes with existing profile. ([#10094](https://github.com/brave/browser-laptop/issues/10094))
+ - Fixed tab previews: stick to changing properties that can be handled by the compositor alone. ([#10291](https://github.com/brave/browser-laptop/issues/10291))
+ - Fixed when the main process crashes, window state is lost. ([#10349](https://github.com/brave/browser-laptop/issues/10349))
+ - Fixed Linux rpm is not signed. ([#10188](https://github.com/brave/browser-laptop/issues/10188))
+ - Fixed minimizing the browser window leaves the tooltip open. ([#8697](https://github.com/brave/browser-laptop/issues/8697))
+ - Fixed extension popup menu items dismissed on click. ([#10130](https://github.com/brave/browser-laptop/issues/10130))
+ - Fixed extension popup not showing. ([#10224](https://github.com/brave/browser-laptop/issues/10224))
+ - Fixed about favicons are taking up way too much space in the session store. ([#10000](https://github.com/brave/browser-laptop/issues/10000))
+ - Fixed js/lib/urlutil.js for safer url parsing. ([#6098](https://github.com/brave/browser-laptop/issues/6098))
```

Comment:
Fixed using chromium URL parsing everywhere in favour of node based URL parsing.